### ISSUE TITLE
Formatage des nombres avec décimales dans la fiche de paie

### ISSUE THREAD

**Issue by mquandalle** [betagouv/mon-entreprise#1994]
Dans la fiche de paie la majorité des nombres sont affichés avec 2 décimales, mais ce n'est pas le cas d'un nombre entier tel que la cotisation pour la complémentaire santé, ce qui casse le bon alignement des nombres :

![image](https://user-images.githubusercontent.com/1730702/152963888-4859e78b-1622-4337-8147-65bd5c027b04.png)

Il me semble que c'est lié aux limitations de l'interface `formatValue` et un besoin à prendre en compte pour retravailler l'API de formatage de publicodes.

**Comment by johangirod** [comment]
Oui, tout à fait, la limitation vient du fait que 'précision' met à jour le maximumNumberOfDigit mais pas le minimum. A garder en tête.
